CHANGELOG.md

```diff
@@ -82,6 +82,7 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 - `variable` condition - now supports variables as both the input and the regular expression
 - `command` event - now suppress console output
 - `cancelconversation` event
+- `time` event - now supports substation and world selection, which made it persistent and static
```

Comment:
What do you guys mean with "substation"? Is this a typo?

Reply:
```suggestion
- `time` event - now supports subtraction and world selection, which made it persistent and static
```